charts/rancher-istio/1.5.901/questions.yaml

```diff
@@ -0,0 +1,3 @@
+labels:
+  rancher.istio.v1.5.901: 1.5.9
+rancher_min_version: 2.4.5-rc1
```

Comment:
We don't need to bump this and cap the previous version ? I'm not actually familiar with changes. Typically unless its security or a very bad bug we don't restrict.

Reply:
I think it depends on the chart type as whether or not you can pick your own version. 

Does Istio allow you to pick what chart you want to deploy?

Reply:
Yes

Reply:
I've approved the changes for the release, but we could discuss how much we want to limit Istio per release. 

In the past, we've limited to only 1 version based on the support matrix but not sure how much that matters/impacts for Istio.

Reply:
Ok, I was unaware of the support matrix limit. I had previously thought we left multiple versions available if they were just patch or minor version changes. 
This change is not an upstream version change, just a bug on our side: https://github.com/rancher/rancher/issues/30964
Given that, I kinda think we should cap the last one.

Reply:
We have decided to cap 1.5.900, putting PR for that in now.